editor/src/messages/tool/common_functionality/shape_editor.rs

```diff
@@ -667,6 +667,12 @@ impl ShapeState {
 					let modification_type = VectorModificationType::SetG1Continuous { handles, enabled: false };
 					responses.add(GraphOperationMessage::Vector { layer, modification_type });
 					continue;
+					if let Some(handles) = point.get_handle_pair(&vector_data) {
+						let modification_type = VectorModificationType::SetG1Continuous { handles, enabled: false };
+						responses.add(GraphOperationMessage::Vector { layer, modification_type });
+					} else {
+						continue;
+					}
```

Comment:
Why have you added this? Seeing as this is after a continue statement, it will never be run.